src/Inventory/Asset/OperatingSystem.php

```diff
@@ -76,6 +76,14 @@ public function prepare(): array
             if (property_exists($this->extra_data['hardware'], 'winprodkey')) {
                 $val->license_number = $this->extra_data['hardware']->winprodkey;
             }
+
+            if (property_exists($this->extra_data['hardware'], 'wincompany')) {
+                $val->company = $this->extra_data['hardware']->wincompany;
+            }
+
+            if (property_exists($this->extra_data['hardware'], 'winowner')) {
+                $val->owner = $this->extra_data['hardware']->winowner;
+            }
```

Comment:
What about `hostid` ? I don't see it imported from the inventory.

Reply:
Indeed, seems it's not handled... Maybe tests are indeed needed

Reply:
> What about `hostid` ? I don't see it imported from the inventory.

Because inventory ```key``` and database ```column name``` are same, so we d'ont need to add ```mapping```

Reply:
HostID does not require conversion, unlike the other 2. The 3 fields are tested in my latest commit.